src/components/pages/HomePage.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import  FeaturedBanners from '../../data/mocks/en-us/featured-banners';
+import  FeaturedProducts from '../../data/mocks/en-us/featured-products';
+import { Main } from '../../styles/grid';
+import { Slider } from '../../components/UI/Slider';
+import { ProductList } from '../products/ProductList';
+
+export const HomePage = () => {
+    const banners = FeaturedBanners.results;
+    const products = FeaturedProducts.results;
```

Comment:
Besides the banners and the products, the categories from the  file "product-categories.json" should also be rendered in the home page.